contracts/bridges/HomeMediator.sol

```diff
@@ -51,21 +51,21 @@ contract HomeMediator {
     // Default payload data length includes the number of bytes of at least one address (20 bytes or 160 bits),
     // value (12 bytes or 96 bits) and the payload size (4 bytes or 32 bits)
     uint256 public constant DEFAULT_DATA_LENGTH = 36;
-    // FX child address on L2 that receives the message across the bridge from the foreign L1 network
-    address public immutable AMBMediator;
+    // AMB Contract Proxy (Home) address on L2 that receives the message across the bridge from the foreign L1 network
+    address public immutable AMBContractProxyHome;
     // Foreign governor address on L1 that is authorized to propagate the transaction execution across the bridge
     address public foreignGovernor;
 
     /// @dev HomeMediator constructor.
-    /// @param _AMBMediator AMB Mediator address (Gnosis).
+    /// @param _AMBContractProxyHome AMB Mediator address (Gnosis).
```

Comment:
Maybe would be better to replace "AMB Mediator" with "AMB Contract Proxy (Home)"